packages/react-scripts/config/webpack.config.js

```diff
@@ -401,6 +401,9 @@ module.exports = function(webpackEnv) {
             {
               test: /\.(js|mjs|jsx|ts|tsx)$/,
               include: paths.appSrc,
+              resolve: {
```

Comment:
Might we have the same issue with other types of modules handled by other loaders? Maybe setting `resolve.symlinks: false` in the top level webpack config would be better?

Reply:
It’s good at least to have this fix to unblock people from use external components in JSX format. For example I’ve a UI library and want to use it in few web projects. I change the UI library code and want to see changes in all projects that use it.

I hope that this solution will help.

Reply:
I'm not sure about using it at the top level as the [docs](https://webpack.js.org/configuration/resolve/#resolvesymlinks) warn about that messing with `npm link`. On second glance though, adding it to `eslint-loader` and the `oneOf` block of loaders should apply it to all of the loaders consistently.

Reply:
Added a commit to resolve symlinks the same way across all loaders.